webpack/.eslintrc

```diff
@@ -26,6 +26,8 @@
       "specialLink": [ "to" ]
     }],
     "promise/prefer-await-to-then": "error",
-    "promise/prefer-await-to-callbacks": "error"
+    "promise/prefer-await-to-callbacks": "error",
+    "no-unused-vars": ["error", { "argsIgnorePattern": "^_" }],
```

Comment:
Allows us to use underscores for unused variables, but still errors otherwise. There are some places I needed this and I think its a good pattern in general that will allow us to be a bit more flexible. It also helps a lot while building a completely new page and not all the pieces are hooked up.